packages/typespec-go/test/armdatabasewatcher/zz_models.go

```diff
@@ -4,7 +4,34 @@
 
 package armdatabasewatcher
 
-import "time"
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"time"
+)
+
+// ArmOperationStatusResourceProvisioningState - Standard Azure Resource Manager operation status response
+type ArmOperationStatusResourceProvisioningState struct {
```

Comment:
This type is showing up due to the removal of option `remove-unreferenced-types=true`, which is expected. All of the ARM *resource types are still omitted.